Sources/swift-format/Frontend/Frontend.swift

```diff
@@ -241,6 +243,61 @@ class Frontend {
       }
     }
 
+    // Load global configuration file
+    // First URLs are created, then they are queried. First match is loaded
+    var configLocations: [URL?] = []
+
+    #if os(Windows)
+      if let localAppData = ProcessInfo.processInfo.environment["LOCALAPPDATA"] {
+        configLocations.append(URL(fileURLWithPath: localAppData))
+      }
+      if let programData = ProcessInfo.processInfo.environment["PROGRAMDATA"] {
+        configLocations.append(URL(fileURLWithPath: programData))
+      }
+    #else
+      if let xdgConfigHome = ProcessInfo.processInfo.environment["XDG_CONFIG_HOME"] {
+        configLocations.append(URL(fileURLWithPath: xdgConfigHome))
+      }
+
+      if let homeLocation = ProcessInfo.processInfo.environment["HOME"] {
+        configLocations.append(URL(fileURLWithPath: homeLocation + "/.config/"))
```

Comment:
Could you use `appendingPathComponent` instead of concatenating strings to form paths?